crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -65,6 +65,22 @@ pub(super) enum EventTimelineItemKind {
     Remote(RemoteEventTimelineItem),
 }
 
+/// A wrapper that can contain either a transaction id, or an event id.
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+pub enum EventItemIdentifier {
```

Comment:
I could like this type, but I'm not sure it's the correct place to get it. `matrix-sdk` seems a better place.

I let @jplatte decides.

Reply:
I don't see why we would move this type into the `matrix-sdk` crate if it's only used inside `-ui`.